WowUp.WPF/AddonProviders/CurseAddonProvider.cs

```diff
@@ -413,7 +413,12 @@ private AddonSearchResult GetAddonSearchResult(CurseSearchResult result, IEnumer
                     DownloadUrl = lf.DownloadUrl,
                     Folders = GetFolderNames(lf),
                     GameVersion = GetGameVersion(lf),
-                    ReleaseDate = lf.FileDate
+                    ReleaseDate = lf.FileDate,
+                    Dependencies = lf.Dependencies.Select(dep => new AddonSearchResultDependency
```

Comment:
Can we add a '?' in case curse decides sometimes its null.

Reply:
resolved